packages/plugin-expo-app/app.js

```diff
@@ -16,23 +16,29 @@ module.exports = {
       lastState = newState
     })
 
-    let nativeBundleVersion, nativeVersionCode
+    let nativeBundleVersion, nativeVersionCode, bundleVersion, versionCode
 
-    if (Constants.appOwnership === 'standalone') {
+    if (Constants.appOwnership !== 'expo') {
       if (Constants.platform.ios) {
         nativeBundleVersion = Application.nativeBuildVersion
+        bundleVersion = Application.nativeBuildVersion
```

Comment:
Rather than make new variables for this with duplicate values, I'd re-use the old ones when adding them to the event/session or rename the old ones to this (but still add them as metadata)

Reply:
^ I'm not sure that makes sense, what I mean is this:

```js
let bundleVersion, versionCode

if (Constants.platform.ios) {
  bundleVersion = Application.nativeBuildVersion
}

if (Constants.platform.android) {
  versionCode = Application.nativeBuildVersion
}

// ...

client.addOnSession(session => {
  session.app.bundleVersion = bundleVersion
  session.app.versionCode = versionCode
})

// ...

client.addOnError(event => {
  event.app.bundleVersion = bundleVersion
  event.app.versionCode = versionCode

  event.addMetadata(
    'app',
    { nativeBundleVersion: bundleVersion, nativeVersionCode: versionCode }
  )
})
```

Reply:
Updated 👍🏾